src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommand.cs

```diff
@@ -55,21 +55,28 @@ internal class InstallToolCommand : CommandBase
             _verbosity = appliedCommand.SingleArgumentOrDefault("verbosity");
 
             var cliFolderPathCalculator = new CliFolderPathCalculator();
-            var offlineFeedPath = new DirectoryPath(cliFolderPathCalculator.CliFallbackFolderPath);
-            _toolPackageObtainer = toolPackageObtainer ?? new ToolPackageObtainer(
-                                       new DirectoryPath(cliFolderPathCalculator.ToolsPackagePath),
-                                       offlineFeedPath,
-                                       () => new DirectoryPath(Path.GetTempPath())
-                                           .WithSubDirectories(Path.GetRandomFileName())
```

Comment:
I  can still change fake project's location right? it is just default?

Reply:
Yes, via the `tempProject` parameter when constructing the installer; this removed behavior is the default it uses if you don't pass in a temp project to use.